saleor/account/forms.py

```diff
@@ -9,6 +10,14 @@
 from .i18n import AddressMetaForm, get_address_form_class
 
 
+class FormWithReCaptcha(forms.BaseForm):
+    def __new__(cls, *args, **kwargs):
+        if settings.ENABLE_RECAPTCHA:
+            cls.base_fields['_captcha'] = ReCaptchaField(
+                label=pgettext_lazy('Captcha field label', 'Are you human?'))
```

Comment:
I'm not a fan of this label, as usually `Recaptcha` is self explanatory

Reply:
What about an empty label?
![screenshot](https://i.imgur.com/EdiHrJA.png)

Reply:
Looks fine to me, I think that recaptcha is labeled as seen on the picture